test/e2e/play_kube_test.go

```diff
@@ -3832,4 +3863,31 @@ ENV OPENJ9_JAVA_OPTIONS=%q
 		Expect(kube).Should(Exit(125))
 	})
 
+	It("podman play kube secret as volume support", func() {
+		err := writeYaml(secretYaml, kubeYaml)
+		Expect(err).To(BeNil())
+
+		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml})
```

Comment:
Are secrets always in a different file or can they be specified alongside the pod/deployment?

Reply:
I haven't tested with a single file, I am not sure of the syntax of including both the pods and the secret within the same yaml, can this be done? All of the examples I saw online specified pre defined secrets in separate yaml files.

Reply:
I see, you are looking for `---` examples? I think those will work, I will add a test using the `generateMultiDocKubeYaml` testing function